src/Microsoft.ML.ImageAnalytics/ImageLoader.cs

```diff
@@ -255,31 +254,30 @@ private Delegate MakeGetterVectorDataViewByteType(DataViewRow input, int iinfo,
                 ValueGetter<VBuffer<byte>> del =
                     (ref VBuffer<byte> dst) =>
                     {
-                        byte[] buffer = null;
-                        if (!_bufferPool.TryTake(out buffer))
-                        {
-                            buffer = new byte[4096];
-                        }
-
                         getSrc(ref src);
-
                         if (src.Length > 0)
                         {
                             string path = src.ToString();
                             if (!string.IsNullOrWhiteSpace(_parent.ImageFolder))
                                 path = Path.Combine(_parent.ImageFolder, path);
-                            if (!TryLoadDataIntoBuffer(path, ref dst, buffer))
-                                throw Host.Except($"Failed to load image {src.ToString()}.");
+                            if (!TryLoadDataIntoBuffer(path, ref dst))
+                            {
+                                var editor = VBufferEditor.Create(ref dst, 0); //Empty Image
+                                dst = editor.Commit();
+                            }
```

Comment:
Love it! This is the right way to handle invalid images and of course ImageClassificationAPI handles this case by dropping it